src/data/finset/lattice.lean

```diff
@@ -358,7 +358,7 @@ begin
   refine ⟨max'_mem _ _, λ t Ht, le_antisymm (le_max' s t Ht) (le_trans a (min'_le s t Ht))⟩,
 end
 
-lemma monotone_max'_min' [decidable_linear_order α] {s : finset α} (hs : s.nonempty) :
+lemma max'_eq_dual_min' {s : finset α} (hs : s.nonempty) :
```

Comment:
Would you mind adding the dual of this lemma (i.e. swapping `min'` and `max'`)?

And in hindsight, it might be better to state this as:
```
min' (image to_dual s) (nonempty.image hs to_dual) = to_dual (max' s hs)
```
and then mark it as simp-lemma.

Reply:
@jcommelin I performed these changes.  Hopefully it will compile now: I believe that the earlier errors came from files that should not have been in this commit!